apps/dashboard/src/app/team/[team_slug]/(team)/~/engine/(instance)/[engineId]/configuration/components/circle-config.tsx

```diff
@@ -0,0 +1,132 @@
+import { FormFieldSetup } from "@/components/blocks/FormFieldSetup";
+import { Spinner } from "@/components/ui/Spinner/Spinner";
+import { Button } from "@/components/ui/button";
+import { Form } from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { TrackedLinkTW } from "@/components/ui/tracked-link";
+import {
+  type EngineInstance,
+  type SetWalletConfigInput,
+  useEngineSetWalletConfig,
+} from "@3rdweb-sdk/react/hooks/useEngine";
+import { useTrack } from "hooks/analytics/useTrack";
+import { useTxNotifications } from "hooks/useTxNotifications";
+import { useForm } from "react-hook-form";
+import { Text } from "tw-components";
+
+interface CircleConfigProps {
+  instance: EngineInstance;
+  authToken: string;
+}
+
+export const CircleConfig: React.FC<CircleConfigProps> = ({
+  instance,
+  authToken,
+}) => {
+  const { mutate: setCircleConfig, isPending } = useEngineSetWalletConfig({
+    instanceUrl: instance.url,
+    authToken,
+  });
+  const trackEvent = useTrack();
+  const { onSuccess, onError } = useTxNotifications(
```

Comment:
Don't use `useTxNotifications` here - its only meant to be used for showing transaction errors/success

Just use `toast.success`, `toast.error` or `toast.promise` directly